packages/js/src/utils/schema.d.ts

```diff
@@ -35,6 +35,7 @@ export interface ExecutorOptions {
   main: string;
   outputPath: string;
   tsConfig: string;
+  swcrcPath?: string;
```

Comment:
This prop should be required since it has a default value, and the function would not work correctly if its null.